docs/topic-guides/blendable-web-apps/view-attaching.md

```diff
@@ -1,42 +1,44 @@
 
-For example, if a user wants to see a profile of a person, the browser makes a request to the People app: `GET http://localhost:8080/people/person/4782`. The response that is sent to the browser will include not only the view from the People app but also views from other apps that are attached to it.
+For example, if a user wants to see a profile of a person, the browser makes a request to the People app: `GET http://localhost:8080/people/person/4782`. The response will include not only the view-model from the People app but also view-models from other apps that are attached to it.
+
+An attachment rule consiststs, of a URI, a token and a context.
 
 On a high level, these are the specific steps involved with sending the response:
 
 1. The browser makes a request for a resource, such as:  `GET http://localhost:8080/people/person/4782`.
 2. The server receives the request and routes it to the corresponding handler in an app \(People\).
-3. The handler is mapped to a token that describes the sort of resource it deals with, in this case, it's the  person table in the database \(`simplified.ring2.person`\). 
-4. The server requests responses from all the other handlers registered in the code host that are mapped to the same token.
-5. For the responses from other apps that are views, the server attaches the views to the view from the initial handler \(which is called the main view\) during the serialization process.
-6. The server sends the response, which now contains views from multiple apps, back to the client.
+3. The handler has an attachment rule that maps the URI to a token that represents the concept, in this case, it's the person table in the database \(`simplified.ring2.person`\). 
+4. The view-model attaching engine requests responses from all the other handlers registered in the code host that have an attachment rule with the same token.
+5. During the serialization process, the server attaches the responses from other apps that are view-models to the main view-model from the initial handler.
+6. The server sends the response, which now contains view-models from multiple apps, back to the client.
 
-By using tokens, the apps don't need to know anything about other apps in the code host - they don't even need to know if there are other apps -  they only have to communicate what concept the handlers deal with. Thus, apps should be built to not depend on, or expect, attaching. 
+By using attchment rules, the apps don't need to know anything about other apps in the code host - they don't even need to know if there are other apps -  they only have to communicate what concept the handlers deal with. The apps should be built to not depend on, but expect, attaching.
```

Comment:
> The apps should be built to not depend on, but expect, attaching.

The meaning of this sentence changed by exchanging "or" for "but". Should apps expect attaching?

Reply:
I think that yes, whenever they use `Self.GET` they need to expect attaching.

Reply:
Okay, then this is fine